### ISSUE TITLE
The Tab component lacks a11y

### ISSUE THREAD

**Issue by petter** [dhis2/ui#79]
The Tab component has been given a style of `outline: none`, without any other alternative focus styling, which makes it really painful to navigate without a mouse. You should consider adding a outlining to it, as you have done with other components, such as the Button.

**Comment by varl** [comment]
https://jira.dhis2.org/browse/LIBS-77
